Source/Core/Common/Timer.cpp

```diff
@@ -118,10 +118,9 @@ void Timer::AddTimeDifference()
 // Get the time elapsed since the Start()
 u64 Timer::GetTimeElapsed()
 {
-  // If we have not started yet, return 1 (because then I don't
-  // have to change the FPS calculation in CoreRerecording.cpp .
```

Comment:
Any idea what this old comment was about?

Reply:
I haven't looked at the history, but I've checked all the current uses of this function and none of them is affected by this change.

Reply:
I checked, the code that used to rely on this is gone.